src/applications/personalization/search-representative/config/chapters/address-change-authorization/index.js

```diff
@@ -1,6 +1,27 @@
+import AuthorizationLimits from '../../../components/AuthorizationLimits';
+
+const authorizationTitle =
```

Comment:
Not worth holding up the PR for but I wonder if we should pull this out into a helper file?

Reply:
I don’t think it’s worth it for this, if we had a quite a few consts like this it might be. Ultimately some of this stuff is subject to change anyways, as down the road the schema stuff will probably need to be pulled out into vets-json-schema anyways

Reply:
Good point. Agreed